packages/opc-timeline/src/opc-timeline.ts

```diff
@@ -0,0 +1,106 @@
+import { LitElement, html, property, customElement } from 'lit-element';
+import style  from './opc-timeline.scss';
+
+@customElement('opc-timeline')
+export class Timeline extends LitElement {
+  @property({ type: Array, attribute: 'steps'}) steps = [];
+  @property({ type: Number, attribute: 'current-step-index' }) currentStepIndex = 0;
+  @property({ type: String, attribute: 'variant'}) variant = 'default';
+  static get styles() {
+    return [ style ];
+  }
+
+  _scrollHandler(direction) {
+    this.shadowRoot.querySelector('#timeline-steps').scrollBy({
+      left: direction === 'left' ?  -900 : 900,
+      behavior: 'smooth'
+    });
+  }
+
+  _getDirectionArrow() {
+    if (this.variant === 'compact') {
+      return {
+        left: html`
+          <span class="timeline__arrow left" @click="${() => {this._scrollHandler('left')}}">
```

Comment:
we shouldn't name classes like this `class="timeline__arrow left"`

it should be `class="timeline__arrow timeline__arrow--left"` or `class="timeline__arrow timeline__arrow--right"`

Reply:
Made the changes